src/index.html

```diff
@@ -2,12 +2,86 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
```

Comment:
can be simplify, use self-closing tags in all cases without content like this `<div class="stars__star" />`

Reply:
![Annotation 2023-08-21 151746](https://github.com/mate-academy/layout_stars/assets/109142598/e1d30d79-3ba0-4d87-82a4-aed6dde99388)
Hi Stas! When I made self-closing tags, I had an error, maybe some of the VS code extensions don't allow me to use this code style, I can remove them, but they were required in the 0 lesson

Reply:
okay, mark as fixed this comment)